image.yaml

```diff
@@ -1,19 +1,19 @@
 schema_version: 1
 name: jboss-datagrid-7/datagrid73-openj9-11-openshift
-version: 1.7
+version: 1.8
 description: "Red Hat JBoss Data Grid 7.3 for OpenShift container image"
 
-from: rh-osbs/ubi8:8-released
+from: registry.redhat.io/ubi8/ubi:latest
 
 labels:
   - name: "com.redhat.component"
     value: "jboss-datagrid-7-datagrid73-openj9-11-openshift-container"
   - name: "org.jboss.product"
     value: "datagrid"
   - name: "org.jboss.product.version"
-    value: "7.3.7.GA"
+    value: "7.3.8.CR1"
```

Comment:
It should be GA for CR artefacts so the final image doesn't contain CR label

Reply:
Applies also for the changes below

Reply:
Updated and contains all changes for x86  (exclude the base images)